src/main/scala/com/github/unchama/seichiassist/subsystems/gachapoint/domain/gachapoint/GachaPoint.scala

```diff
@@ -54,15 +66,20 @@ object GachaPoint {
    *   変換にて得られるガチャ券の総数
    */
   case class Usage(remainingGachaPoint: GachaPoint, gachaTicketCount: Int) {
-    require(gachaTicketCount <= GachaPoint.batchSize, "usage must not exceed batch size")
+    require(gachaTicketCount <= GachaPoint.largeBatchSize, "usage must not exceed batch size")
 
     def asTuple: (GachaPoint, Int) = (remainingGachaPoint, gachaTicketCount)
   }
 
   /**
    * ガチャ券へのポイント交換にて一度に得られるガチャ券の上限
    */
-  final val batchSize = 9 * 64
+  final val largeBatchSize = 9 * 64
```

Comment:
BatchSize についてを扱う以下のような `case class` を追加した後、`largeBatchSize` を `BatchSize` 型としたほうが良いかもしれません。

```scala
case class BatchSize(value: Int) {
  require(value > 0, "batch size must be positive")
}
```

```suggestion
  final val largeBatchSize = BatchSize(9 * 64)
```